package.json

```diff
@@ -20,7 +20,7 @@
     "format": "prettier --write .",
     "test:format": "prettier --check .",
     "test:lint": "yarn workspaces run test:lint",
-    "dockerdev": "docker build --build-arg AXON_VERSION=edge -t axon:local . && docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v wpilib-axon-volume:/usr/src/app/packages/server/data -p 3000:3000 -p 4000:4000 axon:local"
+    "dockerdev": "docker build --build-arg AXON_VERSION=0.0.1 -t axon:local . && docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v wpilib-axon-volume:/usr/src/app/packages/server/data -p 3000:3000 -p 4000:4000 axon:local"
```

Comment:
This means we will be dev testing a released version. I think we want to replace this with edge or local (and build the containers locally too).

Reply:
oops, that is an artifact from debugging